doc/source/tutorials/delaunay-triangulation/assets/delaunay-triangulation.py

```diff
@@ -0,0 +1,30 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+import random
+from scipy.spatial import Delaunay
+
+# Generate a random geometric graph
+random.seed(0)
+g = ig.Graph.GRG(30, 0)
```

Comment:
I think this line deserves an explanation in the comments -- this one basically creates 30 random points in the 2D unit square and creates a graph where the zero distance guarantees that there will be no edges between the graphs.

Wouldn't it be clearer if we created a 30x2 random array with `numpy.random()` and then created a graph later on with `g = Graph(30); g.vs["x"], g.vs["y"] = coords[:, 0], coords[:, 1]`?